src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -413,6 +599,16 @@ export const PatientInfoTable = ({
         handleClickAction={handleConfirmDeletePatient}
         handleClickCancel={handleCancelDeletePatient}
       />
+      <ConfirmModal
+        open={openRestorePatient}
+        title="You are about to restore a patient"
+        body={`Are you sure you want to restore patient #${
+          selectedPatient !== undefined ? selectedPatient.barcodeValue : ''
```

Comment:
i'm confused what this is supposed to achieve - if we don't have `selectedPatient` at this stage, don't we have a lot more problems than just what we're displaying here? in other words is it necessary to do this check?

Reply:
It's there to eliminate possibilities for undefined selectedPatient so TypeScript would not complain.